lib/trino-orc/src/main/java/io/trino/orc/stream/LongInputStreamV1.java

```diff
@@ -63,8 +63,8 @@ private void readValues()
                 throw new OrcCorruptionException(input.getOrcDataSourceId(), "End of stream in RLE Integer");
             }
 
-            // convert from 0 to 255 to -128 to 127 by converting to a signed byte
-            delta = (byte) delta;
+            // convert from 0..255 to -128..127 by simulating a sign extend via shifting
+            delta = delta << 24 >> 24;
```

Comment:
Perhaps that's just limitation of me, but i find this 'too clever' to be used in the code.

Reply:
Yeah, let's not do that. This points to a deficiency or difference in philosophy between ErrorProne and how we want to write code, so it tells me that this check is probably not something we want to enable.

Reply:
Right :) Let's try something else, then (I pushed it already)